packages/eslint-codemod-utils/lib/index.ts

```diff
@@ -93,16 +93,43 @@ export type {
   ImportDefaultSpecifier,
   ImportNamespaceSpecifier,
   ImportSpecifier,
-  ArrayPattern,
   AssignmentOperator,
   AssignmentPattern,
-  ExportAllDeclaration,
+  ArrayPattern,
+  BaseCallExpression,
```

Comment:
Not sure if these `Base*` are really worth exporting / should be exported

Reply:
Aight, removed Base types and alphabetised